content/en/graphing/using_graphs/_index.md

```diff
@@ -0,0 +1,129 @@
+
+### Rollup to aggregate over time
+
+Regardless of the options chosen above, there is always some aggregation of data due to the physical size constraints of the window holding the graph. If a metric is updated every second and you are looking at 4 hours of data, you need 14,400 points to display everything. Each graph displayed has about 300 points shown at any given time.
+
+In the example above, each point displayed on the screen represents 48 data points. In practice, metrics are collected by the Agent every 15-20 seconds. So one day's worth of data is 4,320 data points. You might consider a rollup function that looks at 5 or 10 minutes worth of data for more control over the graph.
+
+To use the rollup function, click the plus sign to the right of the aggregation group and choose `rollup` from the dropdown. Now choose how you want to aggregate the data and the interval in seconds.
+
+To create a single line that represents the total available disk space on average across all machines rolled up in 60-second buckets, you would use a query like this:
+
+{{< img src="graphing/index/references-graphing-rollup-example.png" alt="rollup example" responsive="true" style="width:90%;">}}
+
+When switching to the JSON view, the query looks like this:
+
+```
+"q": "avg:system.disk.free{*}.rollup(avg, 60)"
+```
+
+For more about using the JSON view, see [Graphing Primer using JSON][1].
+
+## Advanced graphing
+
+Depending on your analysis needs, you may choose to apply other mathematical functions to the query. Examples include rates and derivatives, smoothing, and more. See the [list of available functions][7].
+
+The Datadog UI also supports the ability to graph your metrics with various arithmetic operations. Use: `+`, `-`, `/`, `*` to modify the values displayed on your graphs. This syntax allows for both integer values and arithmetic using multiple metrics.
+
+### Metric arithmetic using an Integer
+
+Modify how a metric value is displayed on a graph by performing an arithmetic operation on the metric. For example, to visualize the double of a specific metric, click the **Advanced...** link in the graph editor. Then enter your arithmetic in the `Formula` box, in this case: `a * 2`.
+
+{{< img src="graphing/index/arithmetic_2.png" alt="Arithmetic 2" responsive="true" style="width:75%;" >}}
+
+### Arithmetic between two metrics
+
+To visualize the percentage of a metric by dividing one metric over another. For example:
+
+`jvm.heap_memory / jvm.heap_memory_max`
+
+This can be done in the same manner as above, utilizing the **Advanced...** option in the Graph Editor. From there, select **Add Query**. Each query is assigned a letter: the first metric is represented by **a**, the second metric is represented **b**, and so on.
```

Comment:
```suggestion
This can be done in the same manner as above, utilizing the **Advanced...** option in the graph editor. From there, select **Add Query**. Each query is assigned a letter: the first metric is represented by **a**, the second metric is represented **b**, and so on.
```